modules/Behaviour/behaviour_manage_edit.php

```diff
@@ -171,12 +170,33 @@
                     $column->addLabel('comment', __('Incident'));
                     $column->addTextArea('comment')->setRows(5)->setClass('fullWidth')->setValue($values['comment']);
 
-                //Follow Up
+                $row = $form->addRow()->addHeading('Follow Up', __('Follow Up'));
+
+                //Print old-style followup as first log entry
+                if (!empty($values['followup'])) {
+                    $row = $form->addRow();
+                        $column = $row->addColumn();
+                        $column->addLabel('followUp0', __("Follow Up by {name} at {date}", ['name' => Format::name('', $values['preferredNameCreator'], $values['surnameCreator']), 'date' => Format::dateTimeReadable($values['timestamp'], '%H:%M, %b %d %Y')]));
+                        $column->addContent($values['followup'])->setClass('fullWidth');
+                }
+
+                //Print new-style follow-up as log
+                $behaviourGateway = $container->get(BehaviourGateway::class);
+                $logs = $behaviourGateway->queryFollowUpByBehaviourID($gibbonBehaviourID)->fetchAll();
+
+                if (!empty($logs)) {
+                    $form->addRow()->addContent($page->fetchFromTemplate('ui/discussion.twig.html', [
+                    'discussion' => $logs
+                ]));
+                }
+
+                //Allow entry of fresh followup
                 $row = $form->addRow();
                     $column = $row->addColumn();
-                    $column->addLabel('followup', __('Follow Up'));
-                    $column->addTextArea('followup')->setRows(5)->setClass('fullWidth')->setValue($values['followup']);
-
+                    $column->addLabel('followUp', (empty($logs) ? __('Follow Up') : __('Further Follow Up')) .' / '.__('Notes'))->description(__('If you are the student\'s teacher, please include details such as: the location & lesson, what was the incident, what did you do.'));
```

Comment:
It looks like the description has come across from the first aid feature.
```suggestion
                    $column->addLabel('followUp', (empty($logs) ? __('Follow Up') : __('Further Follow Up')));
```